Enable `zlib-ng-compat` curl feature

This makes ptex fully statically linked; so is desirable.

See here for the current CI error on the M2 shard: https://github.com/a-scie/ptex/actions/runs/3550709980/jobs/5964338449#step:4:34
```
error: failed to run custom build command for `libz-sys v1.1.8`

Caused by:
  process didn't exit successfully: `/Users/jsirois/actions-runner/_work/ptex/ptex/target/debug/build/libz-sys-a98d13fbe368de9e/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=LIBZ_SYS_STATIC
  cargo:rerun-if-changed=build.rs
  CMAKE_TOOLCHAIN_FILE_aarch64-apple-darwin = None
  CMAKE_TOOLCHAIN_FILE_aarch64_apple_darwin = None
  HOST_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_GENERATOR_aarch64-apple-darwin = None
  CMAKE_GENERATOR_aarch64_apple_darwin = None
  HOST_CMAKE_GENERATOR = None
  CMAKE_GENERATOR = None
  CMAKE_PREFIX_PATH_aarch64-apple-darwin = None
  CMAKE_PREFIX_PATH_aarch64_apple_darwin = None
  HOST_CMAKE_PREFIX_PATH = None
  CMAKE_PREFIX_PATH = None
  CMAKE_aarch64-apple-darwin = None
  CMAKE_aarch64_apple_darwin = None
  HOST_CMAKE = None
  CMAKE = None
  running: "cmake" "/Users/jsirois/.cargo/registry/src/github.com-1ecc6299db9ec823/libz-sys-1.1.8/src/zlib-ng" "-DCMAKE_OSX_ARCHITECTURES=arm64" "-DBUILD_SHARED_LIBS=OFF" "-DZLIB_COMPAT=ON" "-DZLIB_ENABLE_TESTS=OFF" "-DWITH_GZFILEOP=ON" "-DCMAKE_INSTALL_PREFIX=/Users/jsirois/actions-runner/_work/ptex/ptex/target/debug/build/libz-sys-04cf1e075848b687/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -arch arm64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -arch arm64" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC -arch arm64" "-DCMAKE_ASM_COMPILER=/usr/bin/cc" "-DCMAKE_BUILD_TYPE=Debug"

  --- stderr
  thread 'main' panicked at '
  failed to execute command: No such file or directory (os error 2)
  is `cmake` not installed?

  build script failed, must exit now', /Users/jsirois/.cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.49/src/lib.rs:1104:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
Error: Process completed with exit code 101.
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 0.1.15
+
+This release completes as much static linking  as possible on Linux by
+bringing zlib into the fold.
+
 ## 0.1.14
 
 This release fixes a bug downloading large files and isolates the ptex

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -5,7 +5,7 @@ members = [
 
 [package]
 name = "ptex"
-version = "0.1.14"
+version = "0.1.15"
 edition = "2021"
 authors = [
   "John Sirois <john.sirois@gmail.com>",
@@ -32,9 +32,7 @@ features = [
   "ssl",
   "static-curl",
   "static-ssl",
-  # TODO(John Sirois): Enable once the M2 runner has CMake installed:
-  #  https://github.com/a-scie/ptex/issues/12
-  #"zlib-ng-compat"
+  "zlib-ng-compat"
 ]
 
 [dev-dependencies]

--- a/RELEASE.md
+++ b/RELEASE.md
@@ -41,3 +41,4 @@ $ git push --tags https://github.com/a-scie/ptex HEAD:main
 The release is automated and will create a GitHub Release page at
 [https://github.com/a-scie/ptex/releases/tag/v&lt;version&gt;](
 https://github.com/a-scie/ptex/releases) with binaries for Linux, Mac and Windows.
+

